lib/index.js

```diff
@@ -10,10 +10,13 @@ var oauth2_client;
  *  @returns {Object} oauth2_client - the Google OAuth2 client
  */
 function create_oauth2_client () {
+  var google_client_id = (OPTIONS.google_client_id) ? OPTIONS.google_client_id : process.env.GOOGLE_CLIENT_ID;
```

Comment:
@ddrager noted. (_makes sense thanks!_)
we would _prefer_ to make the `google_client_id`, `google_client_secret` and `base_url` conform to the **Google Style Guide** for Constants Names:

> **All caps**, **separated with underscores**, declared at the top of the file.

see: https://google.github.io/styleguide/shell.xml#Constants_and_Environment_Variable_Names